### ISSUE TITLE
Ubuntu Linux ENet error.

### ISSUE THREAD

**Issue by zbierajewski** [manicdigger/manicdigger#189]
I get the following error upon starting Manic Digger 2015-02-17

What boggles me is that I have libenet1a installed, and the 2014-08-05 version works flawlessly. 

My Uname: Linux Anthony-Laptop 3.16.0-31-generic #41-Ubuntu SMP Tue Feb 10 15:24:04 UTC 2015 x86_64 x86_64 x86_64 GNU/Linux

The error:
```
3/13/2015 10:34:00 PM: Critical error occurred
ENet.ENetException: The ENet native library failed to initialize. Make sure ENetX86.dll and ENetX64.dll are in the program directory, and that you are running on a x86 or x64-based computer. If you are running on Linux, make sure the libenet.so is in your path. On Ubuntu Linux, install the libenet1a package (1.3.3 or newer) if you haven't already. If you are running on MacOS, make sure libenet.dylib is in your path or program directory.
  at ENet.Native.ENetApi.get_Platform () [0x00000] in <filename unknown>:0 
  at ENet.Native.ENetApi.enet_host_create (ENet.Native.ENetAddress* address, IntPtr peerLimit, IntPtr channelLimit, UInt32 incomingBandwidth, UInt32 outgoingBandwidth) [0x00000] in <filename unknown>:0 
  at ENet.Host.Initialize (System.Net.IPEndPoint address, Int32 peerLimit, Int32 channelLimit, Int32 incomingBandwidth, Int32 outgoingBandwidth) [0x00000] in <filename unknown>:0 
  at GamePlatformNative.EnetHostInitialize (EnetHost host, IPEndPointCi address, Int32 peerLimit, Int32 channelLimit, Int32 incomingBandwidth, Int32 outgoingBandwidth) [0x00000] in <filename unknown>:0 
  at EnetNetClient.Start () [0x00000] in <filename unknown>:0 
  at QueryClient.PerformQuery (System.String ip, Int32 port) [0x00000] in <filename unknown>:0 
  at GamePlatformNative.DownloadServerThumbnail (System.Object o) [0x00000] in <filename unknown>:0
```
-------------------------------

**Comment by croxxx** [comment]
This is caused by a change I made.
Instead of linking against `libenet.so.1` i modified the library to look for `libenet.so`.

Reason for this are the varying package names among different distributions.
On some the package is called `libenet1a`, which is creating just a `libenet.so.1` link. (I just noticed that after last release)
On some others the package seems to be named `libenet2a` which creates a link  `libenet.so.2` and `libenet.so`.

The used library would have to check all combinations of those for it to work properly.
I'll probably have a look into this soon. Any further help appreciated.

**Workaround:**
Create a symlink to the library file present on your system. The name of the symlink should be `libenet.so`
